src/platform/testing/e2e/cypress/support/form-tester/README.md

```diff
@@ -542,11 +550,6 @@ const testConfig = createTestConfig(
 
         // Fill out the rest of the page like normal.
         cy.fillPage();
-
-        // Don't forget to click continue!
```

Comment:
Is the **Continue** click no longer necessary?

Reply:
It's done automatically now. Just forgot to update the doc when that change was made.